front-end/src/app/app.component.ts

```diff
@@ -1,35 +1,126 @@
-import { Component, Input } from '@angular/core';
+import { Component, Input, inject } from '@angular/core';
 import { Task } from 'src/model/Task';
+import { FormControl, FormGroup, ReactiveFormsModule } from '@angular/forms';
+import { CommonModule } from '@angular/common';
+import { TaskCardComponent } from './task-card/task-card.component';
```

Comment:
olyan importok is maradtak bent, amik nincsenek használva. A legegyszerűbb megoldás az lenne, hogy az IDE-ben beállítod, hogy mentéskor törölje a nem használt importokat, így automatikusan törlődni fognak ezek